memory/cloudsql/chat_message_history.go

```diff
@@ -0,0 +1,107 @@
+package cloudsql
+
+import (
+	"context"
+	"errors"
+	"fmt"
+
+	"github.com/tmc/langchaingo/internal/cloudsqlutil"
+)
+
+type ChatMessageHistory struct {
+	engine     cloudsqlutil.PostgresEngine
+	sessionID  string
+	tableName  string
+	schemaName string
+	overwrite  bool
+}
+
+// var _ schema.ChatMessageHistory = &ChatMessageHistory{}
```

Comment:
should this be uncommented?

Reply:
I commented it as rn the struct is not implementing the ChatMessageHistory interface as it is missing some of the required methods, I will remove the comment when it implements the interface.